src/CSSList/CSSList.php

```diff
@@ -169,7 +169,11 @@ private static function parseAtRule(ParserState $oParserState)
                 $sMediaQuery = \trim($oParserState->consumeUntil([';', ParserState::EOF]));
             }
             $oParserState->consumeUntil([';', ParserState::EOF], true, true);
-            return new Import($oLocation, $sMediaQuery ?: null, $iIdentifierLineNum);
+            return new Import(
+                $oLocation,
+                $sMediaQuery !== null && $sMediaQuery !== '' && $sMediaQuery !== '0' ? $sMediaQuery : null,
+                $iIdentifierLineNum
+            );
```

Comment:
I think instead we should initialize `$sMediaQuery` to `''`.  And can and should assume `'0'` to be valid (it might be one day).  (The curious behaviour of PHP casting a string containing the digit zero to false was chosen to make it easiser to handle HTTP request data and database queries, where the number zero would be received as a string.  I don't think that applies here, where it is merely a nonconsequential oversight.)

`Import` (AFAICT) just stores this as a string to be regurgitated later, without having any knowledge of its meaning.  Maybe `Import` should be modified so that this is always a string, with an empty string supplanting `null`.  That would allow for cleaner type declrations.

Reply:
I've added a check for an empty string for the time being (which I guess would match the original intention of the code best). Let's make the types more strict in a separate change.

Reply:
> I've added a check for an empty string for the time being

Neat - makes the code even cleaner.